examples/Cargo.toml

```diff
@@ -13,7 +13,7 @@ rustyline-derive = "0.6"
 scylla = {path = "../scylla", features = ["ssl"]}
 tokio = {version = "1.1.0", features = ["full"]}
 tracing = "0.1.25"
-tracing-subscriber = "0.2.16"
+tracing-subscriber = "0.3.14"
```

Comment:
I'm fine with a bump, but let's add a short rationale why it was added in the commit message - did you miss a feature that's only present in 0.3?

Reply:
The existence of two different versions of a crate caused ambiguity in building book examples. The reason for that is that the book uses `extern crate` specifier, which relies on being provided exactly one crate with matching name.